.node-version

```diff
@@ -1 +1 @@
-10.15.1
+10.15.2
```

Comment:
shouldn't we update `@types/node`?

Reply:
we can, but they don't match exactly usually.  latest looks like 10.12.27?  would you be up for opening in a parallel pr?

Reply:
np 👍